Makefile

```diff
@@ -26,5 +26,11 @@ $(ARCHIVE): $(SCRIPT) .version
 $(SCRIPT):
 	git submodule init && git submodule update
 
+# This is a temporary hack to work around an upstream bug. We want a better
+# way to handle this.
+patch: $(SCRIPT)
+	git submodule foreach --recursive git reset --hard
+	patch -p1 $< < PATCHES/000-fix-grep.patch
```

Comment:
Is it just me or are you applying a reverse diff?  DO you need -r here, or an inversion in 000-fix-grep... ?

Reply:
No, we want to change from `gawk` to `awk`, and from `egrep -o` to `egrep -ao`

Reply:
is not `grep -E` more portable than egrep?

Reply:
We have egrep in the PI, so it's fine. I don't want huge patches, so this is minimal to unbreak it.

Reply:
> No, we want to change from `gawk` to `awk`, and from `egrep -o` to `egrep -ao`

As you explained on slack:  (We want egrep -ao to FAIL so we use awk instead of gawk), I'm now okay with this. Make sure you include testing notes here.